pom.xml

```diff
@@ -59,17 +59,24 @@
     <dependencies>
         <dependency>
             <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure</artifactId>
-            <version>1.9.1-SNAPSHOT</version>
+            <artifactId>adal4j</artifactId>

```

Comment:
track2 probably do not need adal4j lib.

azure-identity uses msal4j

Reply:
Removed import description of `adal4j` from `pom.xml`, and changed the type of exception to `CredentialUnavailableException`.